### PR TITLE
tidy nbs

### PR DESCRIPTION
**Pull request by slemasne**

### CODE REVIEW

File: examples/use-cases/ibor/How do I create holdings in LUSID.ipynb [hunk 1]
```diff
@@ -2,8 +2,24 @@
  "cells": [
```
Comment (MikeMcGarry) [inline examples/use-cases/ibor/How do I create holdings in LUSID.ipynb:2]: <p>Why is there a big gap here now between display_Name and data_type_id?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/179/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Comment (MikeMcGarry) [inline examples/use-cases/ibor/How do I create holdings in LUSID.ipynb:2]: <p>Same here, big gap between lines of code?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/179/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Comment (MikeMcGarry) [inline examples/use-cases/ibor/How do I create holdings in LUSID.ipynb:2]: <p>And here&gt;?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/179/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Comment (MikeMcGarry) [inline examples/use-cases/ibor/How do I create holdings in LUSID.ipynb:2]: <p>Isn't there a LPT method to do this more neatly? See <a href="https://finbourne.atlassian.net/browse/DEV-8580" target="_blank">https://finbourne.atlassian.net/browse/DEV-8580</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/179/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Comment (slemasne) [reply examples/use-cases/ibor/How do I create holdings in LUSID.ipynb:2]: This notebook is paired with the language agnostic article for generating holdings. Therefore we decided to use SDK here as the SDK implementation is the same across all languages.

Comment (slemasne) [reply examples/use-cases/ibor/How do I create holdings in LUSID.ipynb:2]: I don't see gap. 
Let me check with others on the team, so see what they see.

Comment (slemasne) [reply examples/use-cases/ibor/How do I create holdings in LUSID.ipynb:2]: I don't see gap. 
Let me check with others on the team, so see what they see.

Comment (slemasne) [reply examples/use-cases/ibor/How do I create holdings in LUSID.ipynb:2]: I don't see gap. 
Let me check with others on the team, so see what they see.

Comment (rizwansaeed) [reply examples/use-cases/ibor/How do I create holdings in LUSID.ipynb:2]: looks fine to me too

**Pull request merged**